src/components/lib/ConnectWalletDialog.js

```diff
@@ -55,17 +54,15 @@ export const ConnectWalletDialog = ({ shown, onConnected }) => {
 
 	const ChooseWalletStep = () => {
 		return <>
-			<header>
-				<div className="title">Connect To A Wallet</div>
-			</header>
+			<div className="title">Connect to a wallet</div>
 			<p>This wallet will be used to sign transactions on Ethereum.</p>
-
+ 
 			<ul className='wallets'>
 				{
-					WALLETS.map(({ name, icon, showName }) => {
-						return <li className='wallet-option' onClick={() => chooseWallet(name)}>
-							<img src={icon} />
-							{showName && name}
+					WALLETS.map(wallet => {
```

Comment:
Can we extract this
```
<li className='wallet-option' onClick={() => chooseWallet(wallet)}>
 <img src={wallet.icon} />
 {wallet.showName && wallet.name}
</li>
```
 as a separate functional component? I think this will improve readability

Reply:
Addressed in #206 with 81438cb